added The Swedish Reference Genome Portal

annaasklof opened the pull request:
Added info about The Swedish Reference Genome Portal on the reuse page. 
Added The Swedish Reference Genome Portal as a resource (tool).

YvonneKallberg reviewed (approved):
I restructured the page a bit, putting the discovery catalogues in a separate section, what do you think?

YvonneKallberg commented:
Perhaps the header could be better, 'Discover data', 'Data catalogues and registries', or something more 'kärnfullt'?

annaasklof commented:
Good idea, I changed the title to something shorter and 'kärnfullt' as you suggested. I also moved the link to ELIXIR Deposition Databases to the sharing page and added a link from reuse to sharing instead. What do you think?
